ark/utils/data_utils_test.py

```diff
@@ -6,105 +6,62 @@
 import tempfile
 from shutil import rmtree
 
-from mibidata import mibi_image as mi, tiff
+from mibidata import tiff
 
-from ark.utils import data_utils
+from ark.utils import data_utils, test_utils as tu
 import skimage.io as io
 
 
-# required metadata for mibitiff writing (barf)
-METADATA = {
-    'run': '20180703_1234_test', 'date': '2017-09-16T15:26:00',
-    'coordinates': (12345, -67890), 'size': 500., 'slide': '857',
-    'fov_id': 'Point1', 'fov_name': 'R1C3_Tonsil',
-    'folder': 'Point1/RowNumber0/Depth_Profile0',
-    'dwell': 4, 'scans': '0,5', 'aperture': 'B',
-    'instrument': 'MIBIscope1', 'tissue': 'Tonsil',
-    'panel': '20170916_1x', 'mass_offset': 0.1, 'mass_gain': 0.2,
-    'time_resolution': 0.5, 'miscalibrated': False, 'check_reg': False,
-    'filename': '20180703_1234_test', 'description': 'test image',
-    'version': 'alpha',
-}
+def test_load_imgs_from_mibitiff():
 
+    with tempfile.TemporaryDirectory() as temp_dir:
 
-def _create_img_dir(temp_dir, fovs, imgs, img_sub_folder="TIFs", dtype="int8"):
-    tif = np.random.randint(0, 100, 1024 ** 2).reshape((1024, 1024)).astype(dtype)
+        # config test environment
+        fovs = ["Point8_otherinfo", "Point9"]
+        channels = ["HH3", "Membrane", "Other"]
 
-    for fov in fovs:
-        fov_path = os.path.join(temp_dir, fov, img_sub_folder)
-        if not os.path.exists(fov_path):
-            os.makedirs(fov_path)
-        for img in imgs:
-            io.imsave(os.path.join(fov_path, img), tif)
+        filelocs, data_xr = tu.create_paired_xarray_fovs(
```

Comment:
while cute, I don't think tu is that informative, and until the rest of the world starts using our library, probably not a widely recognized abbreviation.

Reply:
second to Noah's comment, I think test_utils would be better